### ISSUE TITLE
Scope

### ISSUE THREAD

**Issue by leouieda** [softwareunderground/subsurface-journal#1]
As pointed out by @filippo82 on Slack, the first thing we should definite is the scope of this be journal.

* What is our target audience?
* How inclusive/exclusive do we want to be?
* What is the niche we're trying to fill?

**Comment by kinverarity1** [comment]
Open access, and therefore, IMO, more inclusive is better.

**Comment by JesperDramsch** [comment]
# Target Audience
- Digital Geoscientists
- Subsurface Engineers
Be it oily, green or anywhere in between, anyone who is analyzing their data through code should be invited. 

# Inclusivity / Exclusivity
I'd love it to be as inclusive as possible but as exclusive as necessary.

In subsurface geo, we often can't share data and I would like to be inclusive of companies legitimately sharing their insights. However, code should definitely be shared to make it reproducible. I'd like to exclude the "advertisement papers" if possible. 

If possible I would envision a tier of:

1. Sharing Code
1. Sharing Models
1. Sharing Data

# Niche
I think our niche could be "Open & Digital" with permissive licensing and open review process.

Generally, just position ourselves as "The fiendly journal".

[Megajournals](https://en.wikipedia.org/wiki/Mega_journal) are increasingly becoming the norm. I'd love to see this become a diverse journal that encourages cross-domain open reproducible publication within Earth Sciences.

**Comment by leouieda** [comment]
> I think our niche could be "Open & Digital" with permissive licensing and open review process.

I thought that was pretty much a given :)  By "niche" I mean: **What are we going to provide that isn't already provided by other journals?** or **What's our elevator pitch?**  With the added restriction of "in (sub)surface science".

**Comment by JesperDramsch** [comment]
@leouieda Is there any journal in geoscience that provides any of this? 😅

But generally be technology friendly would be some sort of niche?

_Sent from my OnePlus 6 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by filippo82** [comment]
Let's take a look at the description of [Computer & Geosciences](https://www.journals.elsevier.com/computers-and-geosciences) and let's forget for a second that it is published by the devil itself:

> Computers & Geosciences publishes high impact, original research at the interface between Computer Sciences and Geosciences. Publications should apply modern computer science paradigms, whether computational or informatics-based, to address problems in the geosciences.

> Computational/informatics elements may include: computational methods; algorithms; data models; database retrieval; information retrieval; near and remote sensing data analysis; data processing; artificial intelligence; computer graphics; computer visualization; programming languages; parallel systems; distributed systems; the World-Wide Web; social media; ontologies; and software engineering.

> Geoscientific topics of interest include: mineralogy; petrology; geochemistry; geomorphology; paleontology; stratigraphy; structural geology; sedimentology; hydrology; hydrogeology; oceanography; atmospheric sciences; climatology; meteorology; geophysics; geomatics; seismology; geodesy; paleogeography; environmental science; soil science; glaciology.

> Other fields may be considered but are not regarded as a priority.

> Computers & Geosciences does not consider:

> Geoscience manuscripts that do not contain a significant computer science innovation. Pure methodological developments (e.g. geophysics, hydrology) are not considered. Pure analytical developments are not considered, unless they have significant implications on computational geoscientific problems.
Computer science manuscripts with no clear application to the geosciences (as defined above).
Manuscripts aiming at solving a geoscientific engineering problem rather than answering a scientific question
Standard code of already well-established, or previously published methods
Graphical User Interfaces (GUIs), unless they provide an original solution to a non-trivial input-handling problem.
Manuscripts that use GIS tools in standard ways

> Code and Data: Computers & Geosciences aims to publish code and supporting data from accepted manuscripts using state-of-the-art technologies. Code should be original and demonstrate a development in research. It should also have clear design and be reproducible, reusable, extensible and maintainable. Manuscripts presenting code, software or implementation of described algorithms need to include a link to a repository where the code can be downloaded. In such cases the open source license should be clearly indicated in submitted manuscripts. Manuscripts that describe code that is not open source are desk rejected. The journal editors offer to fork source code or data repositories that accompany published papers on GitHub https://github.com/CAGEO, to help the community find the author's original repository.

To quote @leouieda,**What we would like to offer that is not already offered by C&G?**

**Comment by bsburnham** [comment]
There is also a new one announced (last Dec. I think) by _that publishing house_ called [Applied Computing and Geosciences](https://www.journals.elsevier.com/applied-computing-and-geosciences/)

**Comment by JesperDramsch** [comment]
> To quote @leouieda,**What we would like to offer that is not already offered by C&G?**

APC below $3200 + tax?

**Comment by filippo82** [comment]
@JesperDramsch that is a very good point but is that the **only** difference?

**Comment by JesperDramsch** [comment]
> @JesperDramsch that is a very good point but is that the **only** difference?

What would *you* like the difference to be?
@filippo82 @bsburnham 

I have stated some points above, I would prefer not to define the scope alone. This is a project from the community. And that actually **is** the main point. Clearly, C&G is **not** an open access journal. And this journal would be a journal serving the community.

I find this funny, but

> Applied Computing & Geosciences publishes original articles, review articles and case studies. Alongside welcoming direct submissions, the journal will benefit from an Article Transfer Service which will allow the author(s) to transfer their manuscript online from Computers & Geosciences thus saving authors time and effort spent on formatting and resubmitting.

on [Applied Computing and Geosciences](https://www.journals.elsevier.com/applied-computing-and-geosciences/) shows, which one is considered the "main journal" and where you can "resubmit".

A [reminder](https://upload.wikimedia.org/wikipedia/commons/2/2b/Welcome%2C_Nature._Seriously_%28from_PLoS%29_%285405189157%29.jpg) that Nature followed PLOS ONE into cross-disciplinary open access publishing. Maybe this could also focus scientific rigour and reproducibility over novelty (as is clearly stated to be the main objective by both C&G and ACP).

**Comment by bsburnham** [comment]
I think first and foremost this should be Open Access.. everyone should be able to read the text, see the code and how the 'science' or 'engineering-magic' was done. This allows people to repeat the same experiments to test the hypothesis on their own data. Scientific Method 101. 

But I also agree on the above suggestions by @JesperDramsch that the journal should be inclusive of any 'Digital Geosciences' .. this would allow people who do digital outcrop work and process/work on the data through code and other digital means, to publish in this journal. Thus it encapsulates subsurface and surface people alike, including quantitative geomorphologists. This journal, I feel, would be attractive to all sorts of people who work on 'Earth' problems through quantitative and programmatic methods. 

- Open Access
- Open to Geos (surface and subsurface) and Engineers who use quantitative/programmtic methods 
- I like the tier system @JesperDramsch mentioned about sharing code, models and/or data

**Comment by joferkington** [comment]
Another consideration is the "bar" for publication.  In many sub-fields, new methods are not considered publishable, even if they're novel.  (e.g. the focus is always on the scientific insight, and the method details can only be published as an appendix to a different paper) Similarly, even in sub-fields where methods are publishable, there's a very high bar for what's considered a publishable method-focused paper.  Small improvements, though often significant, are relegated to institutional knowledge and never published.

I feel there's a need for a journal that encourages publishing small iterations on existing methods and/or new implementations of existing methods.  Given the need for reproducibility and the fact that most methods these days involve software of some sort, this seems like a good fit.

In a nutshell, if C&G does not consider:

- Geoscience manuscripts that do not contain a significant computer science innovation. 
- Pure methodological developments (e.g. geophysics, hydrology) are not considered. 
- Pure analytical developments are not considered, unless they have significant implications on computational geoscientific problems.
- Computer science manuscripts with no clear application to the geosciences (as defined above).
- Manuscripts aiming at solving a geoscientific engineering problem rather than answering a scientific question
- Standard code of already well-established, or previously published methods
- Graphical User Interfaces (GUIs), unless they provide an original solution to a non-trivial input-handling problem.
- Manuscripts that use GIS tools in standard ways

Perhaps this journal should consider some (but not all) of the items above, specifically:

- Pure geoscience methodological developments with accompanying reproducible software/data.
- Pure analytical developments with accompanying reproducible software/data.
- Manuscripts aiming at solving a geoscientific engineering problem rather than answering a scientific question.
- New implementations of already well-established, or previously published methods

**Comment by JesperDramsch** [comment]
> Perhaps this journal should consider some (but not all) of the items above, specifically:

Would reproducible implementations of previously non-reproducible results be of scientific interest?

**Comment by leouieda** [comment]
I really like the points that @joferkington laid out.

To answer "what we can offer that C&G can't", I would say that the main thing are **quality standards**. 

We all know that there is a lot of terrible software published in C&G along with really good things. And most papers don't have their code reviewed at all. All of the comments are about the paper, not the software. This is even true of the Software and Algorithms section in Geophysics.

We can have a really good thing if we offer:

* Cheap/free OA 
* Open review (which tends to lead to friendly and constructive reviews)
* A place for methods, software, and application papers
* Rigorous review of the **full science**: text, code, data, and an honest attempt at replication by the reviewers

**Comment by filippo82** [comment]
@leouieda and others: I have a question that I think it is fundamental to answer before everything else can be decided: **is software/code a mandatory requirement for submitting to this Journal?** In other words (but not the only possible words), **must each submission be accompanied by a software (hosted on a repo) that needs to be reviewed**?

**Comment by leouieda** [comment]
That's an excellent question. I would say **yes, but only if the study required writing software**. Lots of people don't write any software to do their research and should probably not exclude them if the paper is within our scope.

**Comment by leouieda** [comment]
Good point here about what we might consider as novel (ht @filippo82): 
https://twitter.com/scottniekum/status/1104444646275592192?s=19

**Comment by filippo82** [comment]
> That's an excellent question. I would say **yes, but only if the study required writing software**. Lots of people don't write any software to do their research and should probably not exclude them if the paper is within our scope.

@leouieda I'll make a bold statement here: maybe those people are indeed outside our scope. At the end, this Journal proposal comes from the **Software** underground community.

**Comment by bsburnham** [comment]
Great question @filippo82. My two pence on the matter; I do not think we should exclude anyone that doesn't write their own code to do their work/research.

Further to what @leouieda stated, if the Ethos of this journal is an open access place where 'Earth Scientists & Earth Engineers' (maybe I've gone too far with that name..) can submit their work, which is fundamentally quantitative and/or programmatic in nature, then they should be allowed to publish it; even if they **didn't** write any software/code.

IMHO, this Journal should be an alternative (and at the same time something awesome) to the likes of _those-who-shall-not-be-named_ journals and publishing houses for the people like us who do **Digital Geoscience**`.datatype()`. True the journal was first started in **Software** Underground but I think it could be so much more, and allow an avenue for anyone who does quantitative 'geo' work that is transferable across our discipline, both on and below the surface.

**Comment by Japhiolite** [comment]
I agree with @bsburnham for the majority, but ...   
> which is fundamentally quantitative and/or programmatic in nature, then they should be allowed to publish it; even if they didn't write any software/code.  

How would it be handled if licensed / commercial software was used? Would such submission even be handled differently, keeping the _open_ aspect of the journal in mind?

**Comment by filippo82** [comment]
@Japhiolite 

> How would it be handled if licensed / commercial software was used? Would such submission even be handled differently, keeping the _open_ aspect of the journal in mind?

My thoughts:

- If software is part of the submission, then (as mentioned above and in  #2) it shall have an OSI approved licence
- Reviewers should be able to access it and review it

**Comment by JesperDramsch** [comment]
Just wondering how a contribution using proprietary software would be reproducible.

**Comment by leouieda** [comment]
I often struggle with these conflicts and constantly shift depending on how zealous I'm feeling. I get where @bsburnham and @Japhiolite are coming from. But now that I think about it, those people are somewhat well served by the current journals. I would find it hard to distinguish ourselves from JGR, Geophysics, GJI, et al, in that regard. If our edge is going to be on the "open and reproducible" spectrum, then that creates a real conflict with proprietary software (as @JesperDramsch pointed out).

**Comment by Japhiolite** [comment]
Good point, totally forgot about the reproducibility maxim.   
So, some middle ground would be that author do not have to have written their own software / code for their study, provided the used software is not proprietary?

**Comment by bsburnham** [comment]
Indeed reproducibility should be a **key** focus of this. I think using propriety software is fine, but I would agree @leouieda that maybe this type of work would be better suited for the established journals out there (e.g. JGR, JSR etc..). I'm simply thinking of the type of software that is free, open source and can be used by anyone, thus reproducibility is possible. 

For example, [ImageJ Fiji](http://fiji.sc/) is a well-used bit of kit by all manner of people to do image analysis. If someone does a study on tomographic slices of some rock/core, and then a bit of 3D reconstruction, I would say that kind of work _is_ publishable in this Journal. The person should then be required to submit a detailed workflow, thus allowing anyone else to repeat the experiments on their own rock/core. Even if they didn't write any of their image analysis code or use established libraries (e.g. `scikit-image`), they still would have contributed some good quantitative science that is worth sharing with the public.

**Comment by alex-schaaf** [comment]
I totally agree with @bsburnham and @JesperDramsch on the issue of reproducibility - use of open-source software should be an essential requirement for reproducible research, and I think it would be one of the key distinctions of this journal. If people can't access esssential software necessary to carry out the research of a given paper, due to it being stuck behind a paywall (which also tend to be rather steep in subsurface geoscience...), it is virtually irreproducible in my opinion 🤔
